deployment/conf/supported_apps_w_extensions.json

```diff
@@ -1122,8 +1123,13 @@
           "id": "escher_map",
           "title": "EscherMap",
           "app_weight": 1
+        },
+        {
+          "id": "import_specification",
+          "title": "Import Specification",
+          "app_weight": 1
```

Comment:
This file is supposed to be autogenerated IIRC but I don't see any code changes that would cause this to be different

Reply:
IIRC you need to update the mapping here: https://github.com/kbase/staging_service/blob/d41d38c5851350c231950a52d176f9c20a065516/staging_service/autodetect/GenerateMappings.py#L66

Reply:
Well that'd be real cool if that were documented anywhere. I'll add that and fix it.

Reply:
https://github.com/kbase/staging_service/issues/96

Reply:
Added some documentation to the README. Not perfect, but should be enough for now.